src/sentry/replays/query.py

```diff
@@ -153,7 +155,13 @@ def query_replays_dataset(
                 # Make sure we're not too old.
                 Condition(Column("finished_at"), Op.LT, end),
                 # Require non-archived replays.
-                Condition(Column("isArchived"), Op.EQ, 0),
+                BooleanCondition(
+                    BooleanOp.OR,
+                    [
+                        Condition(Column("isArchived"), Op.EQ, 0),
+                        Condition(Column("isArchived"), Op.IS_NULL),
+                    ],
+                ),
```

Comment:
This can be simplified but the two are equivalent.

```
from snuba_sdk.conditions import Or

Or(
    [
        Condition(Column("isArchived"), Op.EQ, 0),
        Condition(Column("isArchived"), Op.IS_NULL),
    ]
)
```

Reply:
Also, if you end up coercing `null` values to `0` we can remove this `is null` check.

Reply:
done!